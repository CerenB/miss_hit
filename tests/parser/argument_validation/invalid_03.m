% From https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html

test_30(ones(2,2,4),144)
% Invalid input argument at position 1. Input must have 2 dimensions
