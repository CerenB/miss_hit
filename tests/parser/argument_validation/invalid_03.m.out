----------------------------------------------------------------------
--  Parse tree for invalid_03.m
Root: Script_File
  Statements: Sequence_Of_Statements
    Statements: Naked_Expression_Statement
      Expression: Reference
        Name: Identifier <test_30>
        Arguments: Reference
          Name: Identifier <ones>
          Arguments: Number_Literal <2>
          Arguments: Number_Literal <2>
          Arguments: Number_Literal <4>
        Arguments: Number_Literal <144>
----------------------------------------------------------------------
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|        ^ info: this token is not linked to the ast
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|             ^ info: this token is not linked to the ast
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|               ^ info: this token is not linked to the ast
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|                 ^ info: this token is not linked to the ast
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|                   ^ info: this token is not linked to the ast
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|                    ^ info: this token is not linked to the ast
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|                        ^ info: this token is not linked to the ast
In invalid_03.m, line 3
| test_30(ones(2,2,4),144);
|                         ^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
